src/Relation/Binary/Bundles.agda

```diff
@@ -11,11 +11,10 @@
 module Relation.Binary.Bundles where
 
 open import Function.Base using (flip)
-open import Level
-open import Relation.Nullary.Negation using (¬_)
-open import Relation.Binary.Core
-open import Relation.Binary.Definitions
-open import Relation.Binary.Structures
+open import Level using (Level; suc; _⊔_)
+open import Relation.Nullary.Negation.Core using (¬_)
+open import Relation.Binary.Core using (Rel)
+open import Relation.Binary.Structures -- most of it
```

Comment:
Should this comment be left in?

Reply:
I added it on purpose. It's for readers of this file, so they know that the lack of `using` is not a mistake, and that a `using` here would be quite "large" and so the ergonomics start to be low. But it still gives information to the reader.

Reply:
Indeed, but I think it's non-conventional as regards existing style guidelines?
Happy to leave it in, but Y(and others')MMV...

Reply:
Blazing new trails, that's me.

Reply:
Yes, but if we invent/uncover new ways to work that we feel useful/appropriate, then we should document them!

Reply:
Waiting for others like @MatthewDaggitt and @Taneb to chyme in one way or another on this.